frontend/src/lib/components/CommandPalette/DebugCHQueries.tsx

```diff
@@ -120,9 +123,25 @@ function DebugCHQueries(): JSX.Element {
                     {
                         title: 'Timestamp',
                         render: (_, item) => (
-                            <span className="font-mono whitespace-pre">
-                                {dayjs.tz(item.timestamp, 'UTC').tz().format().replace('T', '\n')}
-                            </span>
+                            <div className="space-y-2">
+                                <span className="font-mono whitespace-pre">
+                                    {dayjs.tz(item.timestamp, 'UTC').tz().format().replace('T', '\n')}
+                                </span>
+                                {item.queryJson ? (
+                                    <div className="flex">
+                                        <LemonButton type="primary" to={urls.debugQuery(item.queryJson)}>
```

Comment:
Let's make this `targetBlank`, more intuitive than leaving the insight altogether (and safer)